ui/core/src/actions/clp/index.ts

```diff
@@ -139,10 +139,6 @@ export default ({
       asymmetry: string
     ) {
 
-      // Note: Chain needs -1 - 1, 
-      // UI needs to do -10000 - 10000 bc current Fraction floating point limitation
-      asymmetry =  (+asymmetry/10000).toString()
```

Comment:
@iverc This reintroduces error where server request fails because sent out of bounds value:
<img width="309" alt="Screen Shot 2021-02-01 at 2 17 58 PM" src="https://user-images.githubusercontent.com/5235538/106513212-44fa6800-6498-11eb-9e72-0ccf8555f6a5.png">

Reply:
@59023g 
This is remove liquidity test on the backend:
It passes with 1000 assymetry
<img width="358" alt="Screenshot 2021-02-01 at 23 40 56" src="https://user-images.githubusercontent.com/3835515/106521460-f1633b00-64e6-11eb-97d8-129b3842f976.png">

Reply:
@59023g Here is the check for asymmetry on the backend:
`	if m.Asymmetry.GTE(sdk.NewInt(10000)) || m.Asymmetry.LTE(sdk.NewInt(-10000)) {
		return sdkerrors.Wrap(ErrInvalidAsymmetry, m.Asymmetry.String())
	}
	return nil
`

The error message should be updated 
`
	ErrInvalidAsymmetry                = sdkerrors.Register(ModuleName, 9, "Asymmetry has to be 1,-1 or 0")

`

Reply:
So yes, if you send < -10000 or > 10000, it will fail. Fixed error message on the backend side